theme_report/README.md

```diff
@@ -1,20 +1,13 @@
 ## Installation instructions
 
-- install [selenium](https://selenium-python.readthedocs.io/getting-started.html):
-  `pip install selenium`
+- install required libraries
+  `pip install --no-cache-dir -r requirements.txt`
 
-- install beautifulsoup
-  `pip install bs4`
-- Install chromedrive:
-  `brew install chromedriver`
-
-- upgrade chromedrive if needed:
-  `brew upgrade chromedriver`
-
-## Run script
+- Install the required browsers:
+  `playwright install`
 
 - copy env_sample.json to env.json, and add proper values to those env variables:
   `cp env_sample.json env.json`
```

Comment:
I'd include some more comments about what to put in the env.json for the API_KEY and API_URL